supervision/detection/annotate.py

```diff
@@ -258,3 +258,158 @@ def annotate(self, scene: np.ndarray, detections: Detections) -> np.ndarray:
                     thickness=self.thickness,
                 )
         return scene
+
+
+class TriangleAnnotator:
+    """
+    A class for drawing triangles on an image using detections provided.
+
+    Attributes:
+        color (Union[Color, ColorPalette]): The color to draw the triangles,
+            can be a single color or a color palette
+        thickness (int): The thickness of the triangle lines, default is 2
+        text_color (Color): The color of the text on the triangles, default is white
+        text_scale (float): The scale of the text on the triangles, default is 0.5
+        text_thickness (int): The thickness of the text on the triangles,
+            default is 1
+        text_padding (int): The padding around the text on the triangles,
+            default is 10
+
+    """
+
+    def __init__(
```

Comment:
It would be awesome if we could provide some argument to regulate the size and position of the marker.